steamworks_windows.go

```diff
@@ -318,15 +318,6 @@ func SteamUserStats() ISteamUserStats {
 
 type steamUserStats uintptr
 
-func (s steamUserStats) RequestCurrentStats() bool {
```

Comment:
Remove the same function at steamworks_unix.go

Reply:
ok